ale_linters/vim/vint.vim

```diff
@@ -7,40 +8,11 @@ endif
 
 let g:loaded_ale_linters_vim_vint = 1
 
-function! ale_linters#vim#vint#Handle(buffer, lines)
-    " Matches patterns line the following:
-    "
-    " /home/w0rp/.vim/vimrc:198:30: Prefer single quoted strings (see Google VimScript Style Guide (Strings))
-    let pattern = '^.*:\(\d\+\):\(\d\+\): \(.\+\)$'
-    let output = []
-
-    for line in a:lines
-        let l:match = matchlist(line, pattern)
-
-        if len(l:match) == 0
-            continue
-        endif
-
-        let text = l:match[3]
-
-        " vcol is Needed to indicate that the column is a character.
-        call add(output, {
-        \   'bufnr': a:buffer,
-        \   'lnum': l:match[1] + 0,
-        \   'vcol': 0,
-        \   'col': l:match[2] + 0,
-        \   'text': text,
-        \   'type': 'W',
-        \   'nr': -1,
-        \})
-    endfor
-
-    return output
-endfunction
+let s:format = '-f "{file_path}:{line_number}:{column_number}: {severity}: {description} (see {reference})'
```

Comment:
You'll notice that I didn't close the double quote. Looks like `stdin_wrapper` or something else adds an extra double quote.
Try by yourself and you'll understand.